client/src/contexts/AuthContext/useProvideAuth.js

```diff
@@ -29,12 +29,28 @@ const useProvideAuth = () => {
     return user === "User is not in 100Devs"
   }
 
+  // function check if user has needsToBeWelcome property
+  const needsToBeWelcome = () => {
+    return user.needsToBeWelcome;
+  }
+
+  // Delete property of needsToBeWelcome 
+  const deleteNeedsToBeWelcome = () => {
+    const updateUser = { ...user };
+    delete updateUser.needsToBeWelcome;
+    setUser(updateUser)
+    DataService.deleteNeedsToBeWelcome();
+    console.log(user)
```

Comment:
Do we still need this?

Reply:
Yes, that's is the code that is making the delete as soon as deleteNeedsToBeWelcone() is called.